tools/OMIcheck/OMIcheck.ps1

```diff
@@ -1,127 +1,460 @@
-﻿# This script lists the virtual machines in Azure subscriptions of the user,
+﻿# This script lists the virtual machines and scale sets in Azure subscriptions of the user,
 # and looks for vulnerable OMI packages. The report is grouped by subscription and resource group name.
 
 # Set below flag to true if you want vulnerable OMI to be patched.
-# If OMSLinuxAgent is installed, then a minor update will be triggered.
-# If OMI is standalone, the latest bits will be installed from Github.
-$upgradeOMI = $false; #detect only
-$omsVersionGood = "1.13"
+#    - If OMSLinuxAgent or LinuxDiagnostics is installed, then a minor update will be triggered.
+#    - OMI package is installed from Github.
+$upgradeOMI = $false #detect only
+
+$OmiServerGoodVersion = "OMI-1.6.8-1"
+$OmiPkgGoodVersion = "1.6.8.1"
+$LadPkgGoodVersion = "1.5.110-LADmaster.1483"
+$OmsPkgGoodVersion = "1.13.40.0"
```

Comment:
it is `1.13.40-0` not `.0`

Reply:
That's the package version, for example,  dpkg -l | grep omsagent
ii  omsagent                               1.13.40.0